test/com/google/javascript/jscomp/CommandLineRunnerTest.java

```diff
@@ -1139,6 +1139,42 @@ public void testGlobJs2() throws IOException {
     }
   }
 
+  public void testGlobJs3() throws IOException {
```

Comment:
Why not just include FlagUsageException in the `throws` here and then if it's thrown it will still count as a failure.